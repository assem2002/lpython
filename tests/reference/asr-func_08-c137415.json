{
    "basename": "asr-func_08-c137415",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/func_08.py",
    "infile_hash": "7db9398db0036bb22e787d74ea36241f6535928f25e66c09c59e76e3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-func_08-c137415.stderr",
    "stderr_hash": "cec3ab3b772b374fbc85494ed18310b04bd75ebe14390bf7f0071792",
    "returncode": 2
}