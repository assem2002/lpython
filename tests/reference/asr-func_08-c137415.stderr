semantic error: Intent annotation 'In' cannot be used here
 --> tests/errors/func_08.py:3:17
  |
3 | def main0(x: In[In[i32]]):
  |                 ^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
